src/native-common/ScrollView.tsx

```diff
@@ -20,7 +20,7 @@ import ViewBase from './ViewBase';
 //   order for the UI to acknowledge your interaction.
 const overrideKeyboardShouldPersistTaps = RN.Platform.OS === 'macos' || RN.Platform.OS === 'windows';
 export class ScrollView extends ViewBase<RX.Types.ScrollViewProps, RX.Types.Stateless, RN.ScrollView, RX.ScrollView>
-        implements RX.ScrollView {
```

Comment:
This one's a little weird -- I'd expect this to keep the double-indent since it's a continuation of a line that has encapsulation, just like a wrapped function signature.

Reply:
@deregtd I'll check maybe need to add changes to `@typescript-eslint/indent` to handle spaces for `implements`.